test/resources/testproject/src/com/example/five.clj

```diff
@@ -57,7 +57,3 @@
   (let [{:keys [foo bar] :or {foo "foo"}} (hash-map)]
     [:bar :foo]
     (count foo)))
-
-;; This was causing both find-local-symbol and find-macros to blow up, for
-;; different reasons
-::str/bar
```

Comment:
I added it because symbols like that have caused errors in two of our refactorings before.  I don't want a section in the readme about unsupported clojure constructs so I don't want to merge this until this problem is solved.